kore/app/exec/Main.hs

```diff
@@ -793,11 +783,8 @@ koreSerialize execOptions = do
     case outputFileName of
         Nothing -> return (locations, ExitFailure 1)
         Just outputFile -> liftIO $ do
-            execHash <- getExecutablePath >>= Fingerprint.getFileHash
             compact <- compactWithSharing serializedDefinition
-            withFile outputFile WriteMode $ \outputHandle -> do
-                hPut outputHandle (Binary.encode execHash)
-                hPutCompact outputHandle compact
```

Comment:
I included the executable fingerprint check into the `Compact` serialisation (`Binary` instance).